cmd/hecklerd/main.go

```diff
@@ -2735,7 +2741,14 @@ func fetchRepo(conf *HecklerdConf) (*git.Repository, error) {
 			Url:  conf.GitHubHttpProxy,
 		}
 	}
-	remoteUrl := fmt.Sprintf("https://x-access-token:%s@%s/%s/%s", tok, conf.GitHubDomain, conf.RepoOwner, conf.Repo)
```

Comment:
May we Could we move this "empty" check to main function. There is logic to setup default values and checking if values are empty or not. [We can keep all config default/empty check logics at one place.